website/content/docs/server/auth/oidc.mdx

```diff
@@ -59,6 +59,17 @@ You should set the following redirect URLs:
   routable UI address you visit. And note you SHOULD set TLS (`https`) for
   this.
 
+To set the UI-based URL when setting up your auth method, add the `allowed-redirect-uri` flag:
```

Comment:
```suggestion
To set the UI-based URL when setting up your auth method, add the `-allowed-redirect-uri` flag:
```

I did a quick check through some of our other docs and it seems we typically include the `-` when referencing a flag